nixos/modules/services/audio/jmusicbot.nix

```diff
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.jmusicbot;
+in
+{
+  options = {
+    services.jmusicbot = {
+      enable = mkEnableOption "jmusicbot, a Discord music bot that's easy to set up and run yourself!";
+
+      serversettings = mkOption {
+        type = types.path;
+        description = ''
+          Location where config.txt and serversettings.json is saved.
```

Comment:
No declarative configuration?

Reply:
No because I am being lazy and it contains a token which can't easily be removed. Also I could not get it to drop the serversettings.json which is modified at runtime to a different directory than the config.txt even though it has an option for this.